src/IFrameManager.js

```diff
@@ -257,11 +257,12 @@ class IFrameManager {
   }
 
   logDebugMessage(...args) {
-    if (this.debugMode) {
-      const msg = this.isIFrameHost ? "IFrame host" : "IFrame";
-      args.unshift(msg);
+    const msg = this.isIFrameHost ? "IFrame host" : "IFrame";
+    args.unshift(msg);
+    if (process.env.NODE_ENV === "development" && this.debugMode) {
```

Comment:
I think of debugMode as a variable that can be set by the calling provider in any environment. Why should we lock it down to `process.env.NODE_ENV === "development"`?

Reply:
Do we want to enable debugMode in even in production environment? In that case we don't need to enable this conditional.